Tests/NetStandard13.Specs/NetStandard13.Specs.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Chill" Version="4.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="System.Threading" Version="4.3.0" />
```

Comment:
Is this necessary?
According to the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.threading.asynclocal-1?view=netstandard-1.3) `AsyncLocal` is available in `netstandard13`

Reply:
No, fixed.